### DISCUSSION TITLE
Laravel 9.x Livewire not finding new components in custom directory

### DISCUSSION THREAD

**Discussion by Zephni** [livewire/livewire#5607] [1. Help]
I have manually created two files:
The main component: `app\Http\Livewire\UserManagement\UserGroupModal.php`
The blade file: `resources\views\livewire\user-management\user-group-modal.blade.php`

My UserGroupModal looks something like this:
```
<?php
namespace App\Http\Livewire\UserManagement;
// use statements

class UserGroupModal extends LivewireModal
```

But when calling either 
`@livewire('user-management.user-group-modal')` or `@livewire('UserManagement\UserGroupModal')`  in the page that displays it, I am getting the error: **'Unable to find component: [UserManagement\UserGroupModal]'**

Just to note I have tried the following:

1. Clearing all cache with `php artisan cache:clear`, views, optimise, etc
2. Running `php artisan livewire:discover`  (and the bootstrap/cache/livewire-components.php has added the line: `'user-management.user-group-modal' => 'App\\Http\\Livewire\\UserManagement\\UserGroupModal',`)
3. I have tried manually registering the component in AppServiceProvider like so:
```
public function boot()
{
    Livewire::component('user-management.user-group-modal', App\Http\Livewire\UserManagement\UserGroupModal::class);
    // Other stuff
}
```
4. And finally, I asked Bing AI in a final desperation but even they couldn't help me and suggested I post on the forum 😂

I'm really sorry if I've missed something dumb but I can't see anything at the moment as to why this isn't working, any help is greatly appreciated. Or maybe this is just a weird bug?

**Comment by joshhanley** [comment]
@Zephni can you try changing your component to extend Livewire's default component to make sure that there isn't an issue with the modal packaged you are extending?
